DependencyInjection/Configuration.php

```diff
@@ -69,6 +69,26 @@ public function getConfigTreeBuilder()
                     return $v;
                 })
             ->end()
+            ->beforeNormalization() // @deprecated toolbar in 1.3.0, use profiling instead
+                ->ifTrue(function ($v) {
+                    return is_array($v) && array_key_exists('toolbar', $v) && is_array($v['toolbar']);
+                })
+                ->then(function ($v) {
+                    if (array_key_exists('enabled', $v['toolbar']) && 'auto' === $v['toolbar']['enabled']) {
+                        $v['toolbar']['enabled'] = $this->debug; // @deprecated value auto in 1.3.0
+                    }
+
+                    if (array_key_exists('profiling', $v) && is_array($v['profiling'])) {
```

Comment:
i was not specifically talking about the auto thingy, but in general
about default values leading to profiling and toolbar both existing,
even if the user only specified one. but it looks like the only
situation when both exists at this point is when the user actually
specified values for both. in that case we should throw an exception.
its invalid configuration to mix the new and the old name. i see no BC
problem because it means the user just added new configuration that
would have been forbidden with the current release of the bundle, so the
intention was to update the config.